packages/loader/container-loader/src/containerContext.ts

```diff
@@ -157,19 +157,15 @@ export class ContainerContext implements IContainerContext {
         return this._disposed;
     }
 
-    private readonly fluidModuleP = new LazyPromise<IFluidModule>(async () => {
-        const fluidModule = await PerformanceEvent.timedExecAsync(this.logger, { eventName: "CodeLoad" },
-            async () => this.codeLoader.load(this.codeDetails),
-        );
+    public get codeDetails() { return this._codeDetails; }
 
-        return fluidModule;
-    });
+    private readonly _fluidModuleP: Promise<Required<IFluidModuleWithDetails>>;
```

Comment:
>Required [](http://example.com/codeflow?start=44&length=8)

i don;t think you still need the Required here

Reply:
You're right. Not needed.